core/includes/config.inc

```diff
@@ -64,6 +64,11 @@ function config($config_file, $type = 'active') {
  *   The name of the config option within the file to delete. The config option
  *   may contain periods to indicate levels within the config file.
  *
+ * @see config_set()
+ * @see config_get()
+ * @see config_diff()
+ * @see config_menu()
```

Comment:
I don't think `config_diff()` and `config_menu()` are necessary here.

Reply:
I had doubts about adding those two from the very beginning, but thought would be useful to users to have a quick jump links to view them too.

I will remove the links, but I am afraid I have to create a new PR, because the original branch on which this PR was based is gone since long and I forked Backdrop several times over to the same path, so could not restore the branch.